Podfile

```diff
@@ -19,12 +19,14 @@ target 'SDKTest' do
   pod 'DSJSONSchemaValidation'
 end
 
+
 post_install do |installer|
-  installer.generated_projects.each do |project|
-    project.targets.each do |target|
-        target.build_configurations.each do |config|
-            config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '12.0'
-         end
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+    xcconfig_path = config.base_configuration_reference.real_path
+    xcconfig = File.read(xcconfig_path)
+    xcconfig_mod = xcconfig.gsub(/DT_TOOLCHAIN_DIR/, "TOOLCHAIN_DIR")
+    File.open(xcconfig_path, "w") { |file| file << xcconfig_mod }
```

Comment:
Can you explain what these changes are doing?

Reply:
In Xcode 15 there were changes made to build syswtem the changed some of the underlying build system used by cocoapods. Some of the things included renaming part of the file system